Sources/Common/Type/SDKWrapperConfig.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// Metadata about a SDK wrapper/bridge made to use this Customer.io SDK.
+public struct SdkWrapperConfig {
+    /// What is the name of your SDK wrapper?
+    let name: String
```

Comment:
Instead of passing it as a `String`, we should use `Enum` like

```
enum WrapperSource : String {
   case Reactnative = "ReactNative"
   case Flutter // Future = "Flutter"
   case iOS = "iOS"
}
```